src/Microsoft.DotNet.Cli.Utils/BuiltInCommand.cs

```diff
@@ -100,6 +108,29 @@ public ICommand OnErrorLine(Action<string> handler)
             return this;
         }
 
+        private class BuiltInCommandEnvironment : IBuiltInCommandEnvironment
```

Comment:
I think we could move it lower in the future, if we needed to.  But for now, I think it being internal and staying close to where it is used makes sense.